### ISSUE TITLE
integrate with capacity metrics plugin to expose the maintenance status of a circuit as a metric

### ISSUE THREAD

**Issue by dgarros** [nautobot/nautobot-plugin-circuit-maintenance#72]
### Proposed Functionality
It would be very useful to integrate with the [capacity metrics plugin](https://github.com/nautobot/nautobot-plugin-capacity-metrics) to expose the maintenance status of a circuit as a metric. 

The goal is to expose a metric that will reflect the maintenance status of each circuit, something similar to Admin Status and Oper status for interfaces
- `1` circuit is operational
- `2` circuit is in maintenance mode

```
# Circuit operational
circuit_maintenance_status{"circuit": "XXXXX", provider="ntt", circuit_type="peering", site='XX"} 1.0

# Circuit in maintenance mode
circuit_maintenance_status{"circuit": "YYYYYY", provider="ntt", circuit_type="peering", site='YY"} 2.0
```

This is something we are already doing with (few) other plugins, here is an example for the chatops plugin
https://github.com/nautobot/nautobot-plugin-chatops/blob/develop/nautobot_chatops/__init__.py#L59

### Use Case
I can think of 2 main use cases for this feature:
- Visualize in Grafana the maintenance status of a circuit overtime.
- Identify when a circuit is in maintenance to avoid generating an alert, from the alertmanager for example.

**Comment by chadell** [comment]
while working on it, I would only implement the "maintenance mode" as the other is just business as usual:
```
# HELP nautobot_circuit_maintenance_total Active Circuit Maintenances per circuit
# TYPE nautobot_circuit_maintenance_total counter
nautobot_circuit_maintenance_total{circuit="1111111",impact="OUTAGE",circuit_maintenance="123412341234",circuit_type="asdfgfdg",provider="ntt",site="site to define"} 1.0
```

**Comment by chadell** [comment]
Addressed in #73
